modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/ButtonDropDown/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+import ClayDropDown, {Align} from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import {useState} from 'react';
+
+const ButtonDropDown = ({
+	label,
+	align = Align.BottomRight,
+	items,
+	...props
+}) => {
+	const [active, setActive] = useState(false);
+
+	return (
+		<ClayDropDown
+			active={active}
+			alignmentPosition={align}
+			onActiveChange={setActive}
+			trigger={
+				<button className="btn btn-primary">
```

Comment:
Can we use our BaseButton here? It already comes with the icon and everything we need

Reply:
Yeah, sure! Thanks for that. I'll be adding this modification!